public/locales/en/layout/manage.json

```diff
@@ -22,10 +22,11 @@
                 "contribute": "Contribute"
             }
         },
-        "tools": {
-            "title": "Tools",
+        "system": {
+            "title": "System",
             "items": {
-                "docker": "Docker"
+                "docker": "Docker",
+                "log": "Log"
```

Comment:
It’s “logs”, since multiple items are logged

Reply:
No, we only have one log. "Logs" would be if there were multiple logs (eg. error and stdout) - don't you agree?

Reply:
I understand your point, but it shouldn’t be only stdout, should both outputs combined and named logs, that’s what people expect because it’s what is shown from portainer, promox, truenas, docker logs, …